### PR TITLE
Create forkgram.rb

### PR DESCRIPTION
**Pull request by 3xp0rt**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @3xp0rt.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

File: Casks/forkgram-telegram.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 arm:   "6feac1c7089d161f003e921b2d7a4a482b6b69fa3f6a41bbffed4885b63a52f0",
+         intel: "e841defe6ffe755ec94949e6fd188c39f036727807e2cd8b8487e7f9cbf3565d"
+
+  url "https://github.com/Forkgram/tdesktop/releases/download/v#{version}/Forkgram.macOS.no.auto-update_#{arch}.zip"
+  name "Forkgram"
+  desc "Fork of Telegram Desktop"
+  homepage "https://github.com/Forkgram/"
+
+  # Renamed to avoid conflict with telegram
+  app "Telegram.app", target: "Forkgram.app"
```
Comment (SMillerDev) [inline Casks/forkgram-telegram.rb:14]: Why not just declare a conflict?

Comment (3xp0rt) [reply Casks/forkgram-telegram.rb:14]: To be able to use two applications.

**Comment by 3xp0rt** [comment]
> Thank you for your contribution to `homebrew-cask` @3xp0rt.
> 
> Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

Ready

File: Casks/forkgram-telegram.rb (hunk 1 above)
Comment (miccal) [reply Casks/forkgram-telegram.rb:14]: This is also done in another version of `Telegram.app`: https://github.com/Homebrew/homebrew-cask/blob/master/Casks/telegram-desktop.rb

**Comment by miccal** [comment]
Thank you @3xp0rt.

**Review by miccal — APPROVED** [review]

**Pull request merged**